components/log-viewer-webui/client/package-lock.json

```diff
@@ -10559,23 +10772,21 @@
         "colorette": "^2.0.10",
         "compression": "^1.7.4",
         "connect-history-api-fallback": "^2.0.0",
-        "default-gateway": "^6.0.3",
-        "express": "^4.17.3",
+        "express": "^4.19.2",
```

Comment:
This version still seems to be [vulnerable](https://github.com/y-scope/clp/security/dependabot/21).

Reply:
The version that's installed is actually newer. 4.19.2 is actually just the minimum version that they require.

Reply:
Right, my mistake.